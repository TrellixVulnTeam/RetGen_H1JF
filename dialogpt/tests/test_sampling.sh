#!/bin/bash

P=models/762M
MODEL=models/dialogpt/large_ft.pkl
ARGS="--no_token_id"

ROOT=tests/test_sampling_small
DEFAULT_ARGS="--generation_length 20 --fp16 False --use_gpu --top_k 10 --is_sampling sample --seed 42"

exec() {
    CUDA_VISIBLE_DEVICES=0 python run_gpt2.py $DEFAULT_ARGS $ARGS --model_name_or_path $P --load_checkpoint $MODEL --test_file $ROOT.in --output_ref --output_file .sampling.out > $0.log 2> $0.err
    mv $ROOT.sampling.out.resp.txt $0.out
}

exec
