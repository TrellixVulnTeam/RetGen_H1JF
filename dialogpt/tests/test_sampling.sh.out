what is the meaning of life ?	the meaning of life is to live life .	What does this mean ?
what is the meaning of life ?	the meaning of life is to live life .	I would like to hear that question from you and u TheRealCelestein
what is the meaning of life ?	the meaning of life is to live life .	We need answers to that question .
what is the meaning of life ?	the meaning of life is to live life .	I think it's the number of women you can date .
what is the meaning of life ?	the meaning of life is to live life .	I'm Ron Burgundy ?
what is the meaning of life ?	the meaning of life is to live life .	You know that one .
what is the meaning of life ?	the meaning of life is to live life .	To reproduce
what is the meaning of life ?	the meaning of life is to live life .	We don't know ? We are just trying to figure that out .
what is the meaning of life ?	the meaning of life is to live life .	You're the best .
what is the meaning of life ?	the meaning of life is to live life .	The definition
what is the meaning of life ?	the meaning of life is to live life .	The meaning of life is knowing that there is no purpose . Knowing that there is no purpose is not
what is the meaning of life ?	the meaning of life is to live life .	Life is a series of events , events which are not yet known , but are likely to occur one
what is the meaning of life ?	the meaning of life is to live life .	To get laid , presumably .
what is the meaning of life ?	the meaning of life is to live life .	I don't know what the meaning of life is , but the question of what is the meaning of
what is the meaning of life ?	the meaning of life is to live life .	It means death is coming .
what is the meaning of life ?	the meaning of life is to live life .	I am not sure what this sentence means .
what is the meaning of life ?	the meaning of life is to live life .	To pass the time .
what is the meaning of life ?	the meaning of life is to live life .	The question that keeps on asking , is the meaning of life .
what is the meaning of life ?	the meaning of life is to live life .	You mean what I think ?
what is the meaning of life ?	the meaning of life is to live life .	The meaning of life is to live .
what is the meaning of life ?	the meaning of life is to live life .	I think it means the life of God and his son .
what is the meaning of life ?	the meaning of life is to live life .	To pass your time in the afterlife .
what is the meaning of life ?	the meaning of life is to live life .	Life is the universe
what is the meaning of life ?	the meaning of life is to live life .	I believe that is a philosophical question .
what is the meaning of life ?	the meaning of life is to live life .	The definition of life .
what is the meaning of life ?	the meaning of life is to live life .	to find the meaning
what is the meaning of life ?	the meaning of life is to live life .	I don't know , what's it to me ?
what is the meaning of life ?	the meaning of life is to live life .	You and yours and me .
what is the meaning of life ?	the meaning of life is to live life .	Life is what you make of it .
what is the meaning of life ?	the meaning of life is to live life .	It is an answer I don't want to give . I am not sure how to get rid of
what is the meaning of life ?	the meaning of life is to live life .	I don't know . I just know its the end of the world .
what is the meaning of life ?	the meaning of life is to live life .	The meaning of life is to live .
what is the meaning of life ?	the meaning of life is to live life .	That which is life , but in a different way .
what is the meaning of life ?	the meaning of life is to live life .	The universe and its laws .
what is the meaning of life ?	the meaning of life is to live life .	I think that this is the meaning of life , to be as you see it ?
what is the meaning of life ?	the meaning of life is to live life .	To be happy .
what is the meaning of life ?	the meaning of life is to live life .	The universe .
what is the meaning of life ?	the meaning of life is to live life .	The meaning is to get paid
what is the meaning of life ?	the meaning of life is to live life .	I think it's more about the universe in general rather than our particular lives . So , life for
what is the meaning of life ?	the meaning of life is to live life .	To not die .
what is the meaning of life ?	the meaning of life is to live life .	What is love , but a dream .
what is the meaning of life ?	the meaning of life is to live life .	It means nothing .
what is the meaning of life ?	the meaning of life is to live life .	Life is a big question . I don't know for sure .
what is the meaning of life ?	the meaning of life is to live life .	You have to ask yourself the questions that are asked to us as a human .
what is the meaning of life ?	the meaning of life is to live life .	Life as we know it or the universe
what is the meaning of life ?	the meaning of life is to live life .	To be born , you must have been .
what is the meaning of life ?	the meaning of life is to live life .	I'm not sure I know , but if you find out let me know .
what is the meaning of life ?	the meaning of life is to live life .	To be alive
what is the meaning of life ?	the meaning of life is to live life .	I dunno , but I'd say it is a very , very , very , very good thing .
what is the meaning of life ?	the meaning of life is to live life .	What is life ?
